Tests/StreamChatUITests/SnapshotTests/ChatMessageList/ChatMessageListVC_Tests.swift

```diff
@@ -143,9 +144,9 @@ final class ChatMessageListVC_Tests: XCTestCase {
                 cache: nil
             )
             messageDTOWithoutCid.channel = nil
+            mockedMessageWithoutCid = try messageDTOWithoutCid.asModel()
```

Comment:
What is this change affecting?

Reply:
This test was not stable, at least on my local machine; it was causing a crash. So now it should be good 👍